packages/api/tests/endpoints/test-rules.js

```diff
@@ -203,6 +234,8 @@ test('GET gets a rule', async (t) => {
 
 test('When calling the API endpoint to delete an existing rule it does not return the deleted rule', async (t) => {
   const newRule = fakeRuleFactoryV2();
+  newRule.provider = undefined;
+  newRule.collection = undefined;
```

Comment:
Nit: I'd somewhat prefer to do `delete newRule.provider` and `delete newRule.collection`

Reply:
[`717d0e5` (#2012)](https://github.com/nasa/cumulus/pull/2012/commits/717d0e50dc816dd503b437d92a583a563a6c0766)